lib/internal/Magento/Framework/File/Uploader.php

```diff
@@ -207,6 +207,12 @@ public function save($destinationFolder, $newFileName = null)
         $this->_result = false;
         $destinationFile = $destinationFolder;
         $fileName = isset($newFileName) ? $newFileName : $this->_file['name'];
+        $basePath = str_replace('/tmp/','/',$destinationFolder);
+        $tmpFileName = $this->_file['name'];
+        while(file_exists($basePath.'/'.$tmpFileName)) {
+            $tmpFileName = pathinfo($tmpFileName, PATHINFO_FILENAME)."_1.".pathinfo($tmpFileName, PATHINFO_EXTENSION);
+        }
+        $fileName = $tmpFileName;
```

Comment:
There is already such logic in this file: https://github.com/magento/magento2/blob/62d6e69276acfa047ac7e3782a53d52183e4374c/lib/internal/Magento/Framework/File/Uploader.php#L226

Please fix properly.

Reply:
Hello @orlangur 
I have updated code can you review it.